.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+
+# Description
+<!--- Describe your changes to provide context for reviewrs -->
+
+## How to test
+
+<!--- Steps to run to verify the code changes work -->
+
+1.
+1.
+1.
+
+## Issue
+
+closes #
```

Comment:
Same here: I see this section as too specific for PRs with an associated issue. But not all PRs have an associated issue.